src/components/FindWallet/MobileFiltersMenu.tsx

```diff
@@ -0,0 +1,182 @@
+import { useTranslation } from "next-i18next"
+import {
+  Box,
+  Drawer,
+  DrawerBody,
+  DrawerCloseButton,
+  DrawerContent,
+  DrawerHeader,
+  DrawerOverlay,
+  Flex,
+  Text,
+} from "@chakra-ui/react"
+
+import { Button } from "@/components/Buttons"
+
+import { trackCustomEvent } from "@/lib/utils/matomo"
+import { walletsListingCount } from "@/lib/utils/wallets"
+
+import walletData from "@/data/wallets/wallet-data"
+
+import { NAV_BAR_PX_HEIGHT } from "@/lib/constants"
+
+import { FilterBurgerIcon } from "../icons/wallets"
+import OldHeading from "../OldHeading"
+
+import { ResetFiltersButton } from "./ResetFiltersButton"
+import WalletFilterPersona from "./WalletFilterPersona"
+import WalletFilterSidebar, {
+  WalletFilterSidebarProps,
+} from "./WalletFilterSidebar"
+
+import { useWalletTable } from "@/hooks/useWalletTable"
+
+interface MobileFiltersMenuProps extends WalletFilterSidebarProps {
+  showMobileSidebar: boolean
+  onOpen: () => void
+  onClose: () => void
+}
+
+export const MobileFiltersMenu = ({
+  filters,
+  resetWalletFilter,
+  resetFilters,
+  updateFilterOption,
+  updateFilterOptions,
+  selectedPersona,
+  setFilters,
+  setSelectedPersona,
+  showMobileSidebar,
+  onOpen,
+  onClose,
+}: MobileFiltersMenuProps) => {
+  const { t } = useTranslation("page-wallets-find-wallet")
+  const { filteredWallets } = useWalletTable({ filters, t, walletData })
+
+  return (
+    <>
+      <Box
+        display={{ base: "block", lg: "none" }}
```

Comment:
This seems unnecessary. Seems that the parent (find-wallet page) is controlling this already.

Reply:
Fixed